packages/@eventual/core-runtime/src/eventual-factory.ts

```diff
@@ -226,6 +227,20 @@ export function createEventualFromCall(
         );
       },
     };
+  } else if (isSearchCall(call)) {
```

Comment:
Got it. I'll add that. I didn't find the way it works to be intuitive. I'm curious if you had any ideas for how to simplify this machinery?

Reply:
I think it may be possible to centralize all of the commands into one config and make it explicit when a workflow doesn't track. Maybe I'll take a refactor pass when travelling.